packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -60,6 +60,10 @@ export interface RegistrationConfig {
     };
     phone: {
       personAttributeUuid: string;
+      validation?: {
```

Comment:
Sorry for the late review—this all looks good, but this line should be `validation: {`. Config elements will always be defined. I'll include the fix in whatever is my next core PR.

Thanks for this, @donaldkibet !